packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -69,6 +69,7 @@ export const prepareStatusManager = (
   txnsNode,
   {
     marshaller,
+    // eslint-disable-next-line no-unused-vars
     log = makeTracer('Advancer', true),
```

Comment:
this tracer shouldn't call itself `'Advancer'`.
out of scope of this PR, but since you're in the neighborhood...